pygeo/constraints/DVCon.py

```diff
@@ -396,8 +393,8 @@ def writeSurfaceSTL(self, fileName, surfaceName="default", fromDVGeo=None):
         """
         try:
             from stl import mesh
-        except ImportError:
-            raise ImportError("numpy-stl package must be installed")
+        except ImportError as e:
```

Comment:
can you clarify on this fix?

Reply:
So, this is not necessarily a bug, but more of a convenience thing. Usually when we do
```
except someException:
    raise newException(msg)
```
Python will report two exceptions, which is what happens in the code. It will say something like "during the handling of the above exception, <newException> occured". To avoid this, and make Python raise only a single exception, e.g. `newException`, we must instead do
```
except someException as e:
    raise newException(msg) from e
```
then Python understands that the fact that we got `someException` should be masked and not reported.

Reply:
oh that is pretty nice, will try it myself